### PR TITLE
Add leptonica package

### PR DESCRIPTION
**Pull request by diamant3**

### CODE REVIEW

File: leptonica/PSPBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+license=('BSD-2-Clause')
+depends=('zlib' 'libpng' 'jpeg')
+makedepends=()
+optdepends=()
+source=("https://github.com/DanBloomberg/leptonica/releases/download/${pkgver}/$pkgname-${pkgver}.tar.gz")
+sha256sums=("2b3e1254b1cca381e77c819b59ca99774ff43530209b9aeb511e1d46588a64f6")
+
+prepare() {
+    cd "$pkgname-${pkgver}"
+    sed -i 's#@prefix@#${PSPDEV}/psp#' lept.pc.in
```
Comment (sharkwouter) [inline leptonica/PSPBUILD:16]: It seems this one is not working. I'm seeing this in `lept_Release.pc`:
```
prefix=/__w/psp-packages/psp-packages/leptonica/pkg/leptonica/psp
```
Not sure why it's called that either.

Comment (diamant3) [reply leptonica/PSPBUILD:16]: Maybe their CMake is busted, I'll change it to autotools

**Comment by tpimh** [comment]
It seems that there's also another file for CMake: [lept.pc.cmake](https://github.com/DanBloomberg/leptonica/blob/master/lept.pc.cmake).

Used like this:

```cmake
configure_file(lept.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/lept.pc.in @ONLY)
# to resolve generator expression in OUTPUT_NAME
file(
  GENERATE
  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/lept_$<CONFIG>.pc
  INPUT ${CMAKE_CURRENT_BINARY_DIR}/lept.pc.in)

install(FILES ${CMAKE_CURRENT_BINARY_DIR}/lept_$<CONFIG>.pc
        DESTINATION lib/pkgconfig)
```

**Comment by diamant3** [comment]
Thanks. I think that's the issue with the pc filename. Anything I put to `-DCMAKE_BUILD_TYPE=` it results in lept_(input from `-DCMAKE_BUILD_TYPE=`) 😢

**Comment by sharkwouter** [comment]
Thanks!

**Pull request merged**